src/styles/utils/_functions.scss

```diff
@@ -0,0 +1,3 @@
+@function getCatalogWidth ($cardWidth, $cardsNumber, $cardMargin) {
+  @return $cardWidth * $cardsNumber + $cardMargin * ($cardsNumber - 1);
+}
```

Comment:
Your functions use only once in your code. I think you don't need external files for them.